packages/edit-site/src/components/sidebar-edit-mode/sidebar-fixed-bottom.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * WordPress dependencies
+ */
+import { privateApis as componentsPrivateApis } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../private-apis';
+
+const { createPrivateSlotFill } = unlock( componentsPrivateApis );
+const SIDEBAR_FIXED_BOTTOM_SLOT_FILL_NAME = 'SidebarFixedBottom';
+const { Slot: SidebarFixedBottomSlot, Fill: SidebarFixedBottomFill } =
+	createPrivateSlotFill( SIDEBAR_FIXED_BOTTOM_SLOT_FILL_NAME );
+
+export default function SidebarFixedBottom( { children } ) {
+	return (
+		<SidebarFixedBottomFill>
+			<div className="edit-site-sidebar-fixed-bottom-slot">
+				{ children }
+			</div>
+		</SidebarFixedBottomFill>
+	);
+}
+
+export { SidebarFixedBottomSlot };
```

Comment:
No need to export it again.

Reply:
Yes because one's a slot, the other is the fill wrapper. 

Maybe I should rename `function SidebarFixedBottom` to `function SidebarFixedBottomFill` to make it clearer...